src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientScoreSystem.java

```diff
@@ -57,6 +57,10 @@ public String get() {
                 return String.valueOf(redScore);
             }
         });
+        UILabel blackGoalScore = scoreScreen.find("blackGoalScore", UILabel.class);
+        blackGoalScore.setText(Integer.toString(LASUtils.GOAL_SCORE));
+        UILabel redGoalScore = scoreScreen.find("redGoalScore", UILabel.class);
+        redGoalScore.setText(Integer.toString(LASUtils.GOAL_SCORE));
```

Comment:
I am not sure if this is the best solution to go about. It does work, and currently, we do not have an option to change the max amount of points needed in-game, so binding the `UILabel`s with the `GOAL_SCORE` variable is not _that_ necessary atm in my perspective. I would appreciate any opinions on this.

Reply:
Basically the above statement in more detail:
Here I read the two variables (one for Red Team's score and one for Black Team's score) the UILabel that holds the max amount of points you need to score for the game to end, which is set at variable `GOAL_SCORE` within `LASUtils.java`.

What I am doing above, is taking the variable's value directly from `GOAL_SCORE` and giving it to the UILabels, once _(I believe)_ when the Score HUD element is created.

My above comment was about the thought that the above method was not ideal, considering (e.g.) the need for that variable to change sometime in-game - but, I do not think there is any feature that changes that variable in-game, thus no need to use the binding method that the score uses _(see `onScoreUpdateFromServer` in `ClientScoreSystem.java`)_.

Reply:
One option you could go for here is to implement a kind of "getter" for the goal score, that currently returns the value of the `GOAL_SCORE` constant and can - once this is implemented - return the configured goal score instead. Pinging @skaldarnar for second opinion on this option.

Reply:
I think using the constant is a viable first approach here. I just double checked, there is an old `ScoreComponent` which seems to be unused right now. That component could be reused and extended with the target score for winning a game and other configuration. 

That falls into the bucket of cleaning up the package structure of light and shadow and doing a general refactoring/house keeping 🙈